packages/node/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "sourceMap": true,
     "tsBuildInfoFile": "dist/.tsbuildinfo",
     "rootDir": "src",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "strict": true
```

Comment:
Because this is the last package in the repo to enable strict, we can set it on the base tsconfig and remove the explicit option on each of the packages